Sources_v3/WebSocketClient/EventMiddlewares/MemberTypingStateMiddleware.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+struct MemberTypingStateMiddleware<ExtraData: ExtraDataTypes>: EventMiddleware {
+    let database: DatabaseContainer
+    
+    func handle(event: Event, completion: @escaping (Event?) -> Void) {
+        guard let typingEvent = event as? TypingEvent else {
+            completion(event)
+            return
+        }
+        
+        database.write({
+            let dto = MemberDTO.load(
```

Comment:
I suggest to update the `DatabaseSession.loadMember(id, channelId) -> MemberModel` to return `MemberDTO` instead. Then we can user this method here